src/foundry/client/data/documents/fog-exploration.d.mts

```diff
@@ -11,6 +13,22 @@ declare global {
    * The client-side FogExploration document which extends the common BaseFogExploration model.
    */
   class FogExploration extends ClientDocumentMixin(foundry.documents.BaseFogExploration) {
+    /**
+     * Obtain the fog of war exploration progress for a specific Scene and User.
+     * @param query      - Parameters for which FogExploration document is retrieved
+     * @param options    - Additional options passed to DatabaseBackend#get.  Default={}
```

Comment:
Eslint wants escapes on the \. Also, convention is to line break and then mark the `default` with parentheses
```js
* @param options    - Additional options passed to DatabaseBackend#get.
*                                  (default: {})
```

Reply:
Added this convention of our's into the Contributing PR